api/src/main/resources/migrations/surgicalAppointmentAttributeTypes.xml

```diff
@@ -13,12 +13,13 @@
         </preConditions>
         <comment>Adding procedure attribute into surgical_appointment_attribute_type</comment>
         <sql>
+            SELECT user_id into @userid from users where username = 'admin';
```

Comment:
If you are going to use this sort of multiple SQLs, should you not specify "splitStatements" to "false" in the changeset? 
Alternatively, you can just use a single statement with the user_id set as 1. I think just specifying the id to "1" is also fine. since all openmrs/bahmni dbs will come with "Admin" user being the user_id 1.

Reply:
Yes. It makes sense. Referring to other migrations as well. Ex https://github.com/openmrs/openmrs-module-operationtheater/blob/master/api/src/main/resources/liquibase.xml#L324
Hence changed hardcoded user_id '8' to 1